src/realm/bptree.hpp

```diff
@@ -249,9 +249,15 @@ class BpTree : public BpTreeBase {
     BpTree();
     explicit BpTree(BpTreeBase::unattached_tag);
     explicit BpTree(Allocator& alloc);
-    explicit BpTree(std::unique_ptr<Array> init_root)
+    REALM_DEPRECATED("Initialize with ref instead") explicit BpTree(std::unique_ptr<Array> init_root)
```

Comment:
Isn't this used exclusively inside of core? Should we not be able to just delete it?